README.md

```diff
@@ -49,6 +49,28 @@ and then decrypt it on that node. If you like, you may also paste the ciphertext
 into your manifest or Hiera datafiles and then manually invoke the `node_decrypt()`
 function as needed.
 
+## Suitability
+
+Please note that `node_encrypt` is ***not a security panacea***. It will encrypt
+your secrets in the catalog file on disk using the node's certificate, also on
+disk. This means that if an attacker gains root level access to your filesystem,
+then they can likely read both the encrypted secrets and the key used to decrypt
+them.
```

Comment:
Maybe:

```markdown
Please note that `node_encrypt` is ***not a security panacea***. It will encrypt
your secrets in the catalog file on disk using the node's certificate, but the corresponding private key is also on
disk in clear text. This means that if an attacker gains root level access to your filesystem,
then they can likely read both the encrypted secrets and the key required to decrypt
them.
```